packages/client/src/lib.js

```diff
@@ -184,10 +184,10 @@ class Web3Storage {
     }
     const targetSize = maxChunkSize
     const url = new URL('car', endpoint)
-    let headers = Web3Storage.headers(token)
-
-    if (name) {
-      headers = { ...headers, 'X-Name': encodeURIComponent(name) }
+    const headers = {
+      ...Web3Storage.headers(token),
+      'Content-Type': 'application/vnd.ipld.car',
```

Comment:
Ensures we always get a `Content-Type` header even when we do the Node.js fix below.

Reply:
FYI: I have created https://github.com/ipld/js-car/pull/142 that we could reference official mime type, instead of having to copy that string all over.